app/move/controllers/update/personal-details.js

```diff
@@ -0,0 +1,41 @@
+const personService = require('../../../../common/services/person')
+const PersonalDetails = require('../create/personal-details')
+
+const UpdateBase = require('./base')
+
+class UpdatePersonalDetailsController extends UpdateBase {
+  async saveValues(req, res, next) {
+    const person = req.getPerson()
+
+    const identifiers = person.identifiers.map(
+      identifier => identifier.identifier_type
+    )
+    const identifiersData = personService.unformat(person, identifiers)
+    req.form.values = Object.assign({}, identifiersData, req.form.values)
+
+    PersonalDetails.prototype.saveValues.apply(this, [req, res, next])
+  }
+
+  // async saveValues(req, res, next) {
```

Comment:
Just noticed all these comments. Should they be commented out?

Reply:
Surprised it took you this long to find it - it was my canary in the mine ;)